lib/matplotlib/__init__.py

```diff
@@ -215,7 +222,76 @@ def _is_writable_dir(p):
     """
     return os.access(p, os.W_OK) and os.path.isdir(p)
 
+_verbose_msg = """\
+--verbose-LEVEL and the Verbose class are deprecated.
+This functionality is now provided by the standard
+python logging library.  To get more (or less) logging output:"
+    import logging
+    logger = logging.getLogger('matplotlib')
+    logger.set_level(logging.INFO)"""
+
+
+def _set_logger_verbose_level(level_str='silent', file_str='sys.stdout'):
+    """
+    Use a --verbose-LEVEL level to set the logging level:
 
+    """
+    levelmap = {'silent': logging.WARNING, 'helpful': logging.INFO,
+                'debug': logging.DEBUG, 'debug-annoying': logging.DEBUG,
+                'info': logging.INFO, 'warning': logging.WARNING}
+    # Check that current state of logger isn't already more verbose
+    # than the requested level.  If its more verbose, then leave more
+    # verbose.
+    if level_str in levelmap:
+        newlev = levelmap[level_str]
+        oldlev = _log.getEffectiveLevel()
+        if newlev < oldlev:
+            _log.setLevel(newlev)
+            std = {
```

Comment:
Because if someone has set the level inside their downstream code I want that to win over the deprecated command line.

Reply:
But what does that have to do with changing where output goes? Is the default always so high that there's no point to changing output locations?

Reply:
Because they (presumably) have set a logging.Handler as well (or are already happy with the default)

Reply:
Actually, I'm a bit more confused because I can't find any callers of `_set_logger_verbose_level` that set `file_str` to anything but the default.

Reply:
This PR is removing support for `verbose.fileio` matplotlibrc entry, so there should not be a caller.   OTOH, we are still allowing the `--verbose-LEVEL` command line arguments.   

This code could probably go away, but I didn't remove in case someone felt strongly that the matplotlibrc should come back...